src/status_im2/contexts/quo_preview/settings/settings_list.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im2.contexts.quo-preview.settings.settings-list
+  (:require [quo2.components.settings.settings-list.view :as quo]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "Account name:"
+    :key   :title
+    :type  :text}
+   {:label "Chevron:"
+    :key   :chevron
+    :type  :boolean}
+   {:label "Badge:"
+    :key   :badge
+    :type  :boolean}
+   {:label   "Switch:"
+    :key     :switch-props
+    :type    :select
+    :options [{:key   {:value           true
+                       :on-value-change (fn [new-value] (js/alert new-value))}
+               :value "Enable"}]}
```

Comment:
For Switch, Communities, and Button: I would try to make them selectable as booleans in the preview like Chevron & Badge (instead of `select option, enable, clear, close`. There should be some examples of how to do this in other preview namespaces.

Reply:
Yeah, agree, I search for some example but didn't find any, will search again to see if I have better luck. I tried to do it on my own but it didn't work.

Reply:
@rfahernandez The input component looks like a good example:

The `:button` boolean is set in preview, and the button properties are `assoc`'ed if `true`: https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/quo_preview/inputs/input.cljs#L96

Reply:
Oh, sorry, I've use the approach used on https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/quo_preview/switcher/switcher_cards.cljs
Where the preview call `get-mock-data` and sets a value when the prop is a truly  value, in this case is true because the boolean selector. Is that ok or would you prefer the input approach?